util/src/com/thoughtworks/go/util/LogFixture.java

```diff
@@ -25,56 +25,38 @@
 import org.apache.log4j.Logger;
 import org.apache.log4j.spi.LoggingEvent;
 
-public class LogFixture extends AppenderSkeleton {
-
-    public static LogFixture startListening() {
-        return startListening(Level.DEBUG);
-    }
-
-    public static LogFixture startListening(Level level) {
-        LogFixture testing = new LogFixture();
-        testing.activateOptions();
-        setLevel(level);
-        Logger.getRootLogger().addAppender(testing);
-        return testing;
```

Comment:
If I understand this "fix" properly, we were basically creating a new appender for each test, which you're now auto-closing. Plus, you're creating a new logger for each test.

Would it be possible to just have `LogFixture` take in a level, and auto-generate the logger name, so it does not need to be passed in, even if just to avoid copy-paste errors?